### DISCUSSION TITLE
asus 40hx cmp 8G，Graphics card，How to optimize settings

### DISCUSSION THREAD

**Discussion by woaizca** [lllyasviel/Fooocus#2758] [Q&A]
When I am using stable diffusion, I use two settings as follows:
1. "D: \ sd webui aki \ webui user. bat", set COMMANDLINE.ARGS=-- allow code -- xforms -- skip torch cuda test -- no hall vae -- API -- no hall -- no flashing -- disable can check -- medvram -- theme=light
2. "D: \ sd webui aki \ modules \ devices. py",,
Def auto cast (disable=False):
If disable:
Return contextlib. nullcontext()
If dtype==torch. float32 or shared. cmd_ops. precision=="full":
Return contextlib. nullcontext()
Return torch. autocast ("cuda", dtype=torch. float32, enabled=True) "------------------------《 Especially this sentence》
When I use SDXL for stable diffusion, the drawing time is 2 minutes, but when I use FOOOCUS, the drawing time is 8 minutes. How can I set the parameters to shorten the drawing time of FOOOCUS.

**Comment by mashb1t** [comment]
Please provide the full terminal output for Fooocus to be able to give you qualified advice.
